src/documentation/runners/apex.md

```diff
@@ -5,5 +5,70 @@ permalink: /documentation/runners/apex/
 ---
 # Using the Apache Apex Runner
 
-This page is under construction ([BEAM-825](https://issues.apache.org/jira/browse/BEAM-825)).
+The Apex Runner executes Apache Beam pipelines using [Apache Apex](http://apex.apache.org/) as an underlying engine. The runner has broad support for the [Beam model and supports streaming and batch pipelines]({{ site.baseurl }}/documentation/runners/capability-matrix/).
 
+[Apache Apex](http://apex.apache.org/) is a stream processing platform and framework for low-latency, high-throughput and fault-tolerant analytics applications on Apache Hadoop. Apex has a unified streaming architecture and can be used for real-time and batch processing. With its stateful stream processing architecture, Apex can support all of the concepts in the Beam model (event time, triggers, watermarks etc.).
+
+
+## Apex-Runner prerequisites and setup
+
+You may set up your own Hadoop cluster,  and [setup Apache Apex on top of it](http://apex.apache.org/docs/apex/apex_development_setup/) or choose any vendor-specific distribution that includes Hadoop and Apex pre-installed. Please see the [distribution information on the Apache Apex website](http://apex.apache.org/downloads.html).
+
+Wordcount example to run on Apex-Runner
+```
+git clone https://github.com/tweise/apex-samples.git
+pushd apex-samples/beam-apex-wordcount
+mvn clean install -DskipTests
+```
+
+## Running wordcount using Apex-Runner
+Currently the wordcount program reads data from /tmp/input/ and stores results into /tmp/output/ on HDFS.
+
+Download some data for processing and put it on HDFS
+```
+curl http://www.gutenberg.org/cache/epub/1128/pg1128.txt > /tmp/kinglear.txt
```

Comment:
The default is to use gs://<file-name> but we are using HDFS based input. So added steps explicitly for this.